container/service_options.go

```diff
@@ -71,7 +74,12 @@ func (options *ServiceOptions) swarmPorts() (ports []swarm.PortConfig) {
 	return
 }
 
-func (options *ServiceOptions) swarmMounts() (mounts []mount.Mount) {
+func (options *ServiceOptions) swarmMounts(force bool) (mounts []mount.Mount) {
+	// hack for preventing mount when in CircleCI
```

Comment:
A bit hacky, I would add a comment TOFIX just to make sure that we can find this again when the CI allow to mount directories

Reply:
Comment updated